art-blocks-api/api-overview.md

```diff
@@ -192,31 +190,81 @@ Subgraph information is serviced by a decentralized group of server operators ca
 
 The Art Blocks testnet subgraph can be queried at the URL below:
 
-| The Graph Service | Art Blocks Data | URL |
-| --- | --- | --- |
-| Hosted Service | Yes | https://thegraph.com/hosted-service/subgraph/artblocks/art-blocks-artist-staging-goerli |
+| The Graph Service | Art Blocks Data | URL                                                                                     |
+| ----------------- | --------------- | --------------------------------------------------------------------------------------- |
+| Hosted Service    | Yes             | https://thegraph.com/hosted-service/subgraph/artblocks/art-blocks-artist-staging-goerli |
 
 The Art Blocks Arbitrum subgraphs can be queried at the URLs below:
 
-| The Graph Service | Art Blocks Data | Environment | URL |
-| --- | --- | --- | --- |
-| Hosted Service | Yes | Production | https://thegraph.com/hosted-service/subgraph/artblocks/art-blocks-arbitrum |
-| Hosted Service | Yes | Staging | https://thegraph.com/hosted-service/subgraph/artblocks/art-blocks-staging-arbitrum |
+| The Graph Service | Art Blocks Data | Environment | URL                                                                                |
+| ----------------- | --------------- | ----------- | ---------------------------------------------------------------------------------- |
+| Hosted Service    | Yes             | Production  | https://thegraph.com/hosted-service/subgraph/artblocks/art-blocks-arbitrum         |
+| Hosted Service    | Yes             | Staging     | https://thegraph.com/hosted-service/subgraph/artblocks/art-blocks-staging-arbitrum |
 
 **Recommendation:** Using the above links, familiarize yourself with the subgraph’s schema, via the GraphQL playground.
 
-<br>
+### Getting Minter Details
+
```

Comment:
IIUC, this information can also be queried from our Hasura GraphQL API in addition to subgraph directly – could we highlight that more prominently here?

Reply:
good call - I've highlighted more prominently in [055c860](https://github.com/ArtBlocks/artblocks-docs/pull/282/commits/055c860b9b88740e2852c3397fbc5584ef6537c5), including some extra details about what to query in Hasura vs. Subgraph